pallets/schemas/src/migration/v4.rs

```diff
@@ -14,21 +14,23 @@ use sp_runtime::TryRuntimeError;
 use sp_std::{collections::btree_map::BTreeMap, vec::Vec};
 
 /// get known schema names for mainnet
-pub fn get_known_schemas<T: Config>() -> BTreeMap<SchemaId, Vec<u8>> {
+pub fn get_known_schemas<T: Config>() -> BTreeMap<SchemaId, (Vec<u8>, bool)> {
```

Comment:
if you compile the migration with try runtime to check against the mainnet. It will not compile

Reply:
might need to modify `pre_upgrade` and `post_upgrade`